src/mysql.erl

```diff
@@ -729,10 +733,13 @@ handle_call(start_transaction, _From,
                                                ?cmd_timeout),
     SockMod:setopts(Socket, [{active, once}]),
     State1 = update_state(Res, State),
-    {reply, ok, State1#state{transaction_level = L + 1}};
-handle_call(rollback, _From, State = #state{socket = Socket, sockmod = SockMod,
-                                            status = Status, transaction_level = L})
+    {reply, ok, State1#state{transaction_level = L + 1, monitors = [{FromPid, MRef} | Monitors]}};
+handle_call(rollback, {FromPid, _}, State = #state{socket = Socket, sockmod = SockMod,
+                                            status = Status, transaction_level = L,
+                                            monitors = [{FromPid, MRef}|NewMonitors]})
```

Comment:
Indentation is broken here because FromPid was added.

Reply:
Fixed